cgsmiles/drawing.py

```diff
@@ -28,6 +37,12 @@ def draw_molecule(graph,
     if not ax:
         fig, ax = plt.subplots(1,1)
 
+    # scale edge widths and fontsize
+    if scale:
+        edge_widths = edge_widths / scale
+        mapped_edge_width = mapped_edge_width / scale
+        fontsize = fontsize / scale
```

Comment:
Can you explain what's happening with `scale` and what it has to do with the bounding box of the plot?

Reply:
does not do anything to the bonding box itself; it only scales the image dimensions. The plotter does not automatically wrap an image into the canvas. The default settings are chosen such that any reasonable canvas size will work. Scale can be used to adjust the size if needed. This strategy is the only option I could implement that ensures the same setting (scale, default_bond) gives the same-sized molecule images independent of the canvas size.